services/users/src/main/java/org/opfab/users/configuration/oauth2/WebSecurityConfiguration.java

```diff
@@ -114,8 +114,7 @@ public static void configureCommon(final HttpSecurity http) throws Exception {
 
 
     public static AuthorizationManager<RequestAuthorizationContext> authenticatedAndIpAllowed() {
```

Comment:
Remove "ipAllowed" from methos names
Not needed to call AuthorizationManagers.allOf(..)  when there is just one AuthorizationManager